frontend/src/metabase/components/sortable/sortable.module.css

```diff
@@ -1,3 +0,0 @@
-:global(.react-sortable-hoc-helper) {
```

Comment:
It turned out `react-sortable-hoc` isn't used anymore so these CSS files could be deleted